pom.xml

```diff
@@ -205,6 +205,11 @@
     </profile>
     <profile>
       <id>rat</id>
+      <activation>
```

Comment:
I would suggest using `<activeByDefault>true</activeByDefault>` instead of depending on a not-existing property. I would be more obvious what's happening.

Reply:
Yes, you are right. This is simpler and cleaner.